server/views/partials/varyCallToActions.njk

```diff
@@ -69,7 +69,7 @@
                     href: "/licence/vary/caseload",
                     attributes: { 'data-qa': 'return-to-caselist' }
                 }) }}
-
+            
```

Comment:
Can be ignored if there are no other changes in the PR, but presumably this can be removed

Reply:
Oooo added as we were missing a line between these cases, just following consistency with where we have a line between each other case in the file.